governance.md

```diff
@@ -1,14 +1,88 @@
-# Tekton Governance
+# Tekton Bootstrap Governance
 
-The Tekton project was donated to the [CDF](cd.foundation) in March 2019.
-Establishing governance is still a work in progress and for now, Tekton is
-overseen by the Knative Build WG.
+The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
 
-## Procedures
+Members are:
 
-### Creating a new repository
+* [Andrew Bayer](https://github.com/abayer) (CloudBees)
+* [Vincent Demeester](https://github.com/vdemeester) (RedHat)
```

Comment:
(absolute) nit: `s/RedHat/Red Hat/g`

Reply:
Haha yeah, forgot the hold. I'll get that updated.